src/controller/CHIPDeviceController.cpp

```diff
@@ -540,6 +540,22 @@ CommissioneeDeviceProxy * DeviceCommissioner::FindCommissioneeDevice(NodeId id)
     return foundDevice;
 }
 
+CommissioneeDeviceProxy * DeviceCommissioner::FindCommissioneeDevice(const Transport::PeerAddress & peerAddress)
+{
+    // Q: can we distinguish BLE sessions?
```

Comment:
Is the question whether we can distinguish BLE sessions from non-BLE sessions, or whether we can distinguish BLE sessions from one another.

I believe we have a special 'BLE' peer address, so we can tell if a session is BLE.  But all BLE sessions will have the same 'BLE' peer address, so can't be distinguished from one another.

Reply:
Sorry, this was removed in a previous version of this PR. Will remove again. We can't distinguish BLE sessions right now, but we are also incapable of opening > 1.